app/controllers/course/forum/posts_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Course::Forum::PostsController < Course::Forum::Controller
+  before_action :load_topic
+  authorize_resource :topic
+
+  include Course::Discussion::PostsConcern
+
+  def create
+    if super
+      redirect_to course_forum_topic_path(current_course, @forum, @topic),
```

Comment:
I *think* when you implement the other controllers you will not be reusing this, but okay refactor later.